src/services/service-locator.ts

```
@@ -0,0 +1,17 @@
+const serviceRegistry = new Map<Function, any>();
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)